pkg/templates/externalccm/hetzner.go

```diff
@@ -59,7 +59,7 @@ func ensureHetzner(ctx *util.Context) error {
 	dep := hetznerDeployment()
 	want, err := semver.NewConstraint("<= " + hetznerCCMVersion)
 	if err != nil {
-		return errors.Wrap(err, "failed to parse digital CCM version constraint")
```

Comment:
This should probably be changed for DigitalOcean to says `digitalocean` instead of `digital`. :)

Reply:
done in pkg/templates/externalccm/digitalocean.go